frontend/mono_frontend/src/components/JarsListView.jsx

```diff
@@ -0,0 +1,51 @@
+import { Link } from 'react-router-dom'
+import { JarIcon } from './JarIcon'
+import { ProgressBar } from './ProgressBar'
+
+export const JarsListView = ({ jarsData }) => {
+	let totalSum = null
+
+	if (jarsData) {
+		totalSum = jarsData.reduce((sum, jar) => sum + jar.balance / 100, 0)
+		totalSum = Number(totalSum).toFixed(2)
+	}
+
+	const calcTotalProc = function (balance, goal) {
+		return String(Math.round((balance / goal) * 100) + '%')
+	}
+
+	return (
+		<div className='jars'>
+			<div className='jars__box'>
+				<h1 className='jars__title'>Jars</h1>
+				<p className='jars__textMoney'>
+					Total balance in all your JARS: {totalSum} ₴
+				</p>
+				<ul className='jars__list'>
+					{jarsData.map(jar => (
+						<li className='jars__item'>
```

Comment:
IMO we can store single jar list item in separate component to reduce some visual pressure

Reply:
Done